uitools/import/Esri/ArcGISRuntime/Toolkit/Callout.qml

```diff
@@ -280,7 +280,7 @@ Item {
     Connections {
         id: calloutConnection
         target: calloutData
-        onScreenPointChanged: {
```

Comment:
good fix

Reply:
I didn't change this, so must be someone else's good work.

Reply:
@jared-2016 💪